domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3236,7 +3270,7 @@ public void selectSingle(AsyncResult<SQLConnection> conn, String sql, Handler<As
   }
 
   /**
-   * Run a parameterized/prepared select query with the readonly connection
+   * Run a parameterized/prepared select query
    * and return the first record, or null if there is no result.
    *
    * <p>To update see {@link #execute(String, Handler)}.
```

Comment:
This line is misleading. It should be removed and this new line been appended at the end of the javadoc:

```
   * @see #selectSingleRead(String, Tuple, Handler)
```